examples/controls/app.js

```diff
@@ -1,13 +1,14 @@
 /* global window */
 import React, {Component} from 'react';
 import {render} from 'react-dom';
-import MapGL, {Marker} from 'react-map-gl';
+import MapGL, {Marker, Popup} from 'react-map-gl';
 
 import CityPin from './components/city-pin';
 
 import CITIES from './data/cities.json';
 
 import './app.css';
+import 'mapbox-gl/dist/mapbox-gl.css';
```

Comment:
Echoing the other diff, this only works for webpack? We should add a `note` here saying that if that's the case and we want to continue using this method.